<launch>
  <node respawn="true" pkg="joy"
        type="joy_node" name="turtle_joy" >
    <param name="dev" type="string" value="/dev/input/js0" />
    <param name="deadzone" value="0.12" />
  </node>
  <!-- keyboard teleoperation node for omo r1 robot -->
  <arg name="set_multi_robot_name" default=""/>
  <node pkg="omo_r1mini_teleop" type="omo_r1mini_teleop_joy.py" name="omo_r1mini_teleop_joystick"  output="screen">
    <param name="max_fwd_vel" value="0.3" />
    <param name="max_rev_vel" value="0.25" />
    <param name="max_ang_vel" value="2.0" />
    <param name="tf_prefix" value="$(arg set_multi_robot_name)"/>
  </node>
</launch>
